pkg/autometrics/prometheus/prometheus.go

```diff
@@ -60,29 +76,45 @@ const (
 // Make sure that all the latency targets you want to use for SLOs are
 // present in the histogramBuckets array, otherwise the alerts will fail
 // to work (they will never trigger.)
-func Init(reg *prometheus.Registry, histogramBuckets []float64) error {
+func Init(reg *prometheus.Registry, histogramBuckets []float64, buildInformation BuildInfo) error {
+	autometrics.SetCommit(buildInformation.Commit)
+	autometrics.SetVersion(buildInformation.Version)
+	autometrics.SetBuildTime(buildInformation.BuildTime)
```

Comment:
For what it's worth, I left out build time because I wasn't sure how we'd actually use that information. I think it's fine to keep in but for now it feels like extraneous data